AlumniApp/core/CreateAccount.js

```diff
@@ -24,107 +30,78 @@ export default class CreateAccount extends React.Component{
 		this.state ={
 			fullName: '',
 			email:'',
-			password: '',
-			confirmPassword:''
-		}
+			password: ''
+		};
+		
+		this.onPasswordVerification = this.onPasswordVerification.bind(this);
+		this.onPasswordConfirm = this.onPasswordConfirm.bind(this);
 	}
-<<<<<<< HEAD
-
-}
-
-/**
- *
- * Renders the fields
- * Have the User input Name, E-mail, Password, Confirm Password
- * Submit button changes state when it is pressed
- */
-render(){
-	return(
-		<View>
-			<TextInput
-				placeholder="Name"
-				onChangeText ={(text) => this.setState({Name: text})}
-				value{(this.state.name)}
-				maxLength = {30}
-			/>
-			
-			<TextInput
-				placeholder="E-mail"
-				onChangeText ={(text)=>this.setState({email: text})}
-				value{(this.state.email)}
-				maxLength = {30}
-			/>
-			<TextInput
-				<Text>Please input a password at least 8 characters</Text>
-				<Text>To help make the password stronger use special characters or numbers</Text>
-				placeholder="Password"
-				onChangeText={(text)=>this.setState({password: text})
-				secureTextEntry={true}
-				maxLength = {30}
-				<ReactPasswordStrength
```

Comment:
This was a confusing line since it *seems* like the intent was to make the `TextInput` do Password Strength tests but placing an element inside the attribute section of another element is not valid.

The solution I chose was to re-implement the functionality of ReactPasswordStrength to some degree. This is not a port but it has similar capabilities.